spring/src/main/java/io/cucumber/spring/CucumberSpringTest.java

```diff
@@ -0,0 +1,14 @@
+package io.cucumber.spring;
+
+import org.apiguardian.api.API;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@API(status = API.Status.STABLE)
+public @interface CucumberSpringTest {
```

Comment:
I suggested `@CucumberSpringTest` but I'm not sure if that is the right name. This annotation allows a user to mark a application context configuration class as the class that Cucumber should use. Maybe different name can convey this better.

Reply:
What about `@CucumberContextConfiguration` ?

Reply:
That would work.